code-challenges.js

```diff
@@ -32,7 +32,24 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
-// Pseudo code:
+// Pseudo code: we need to create a function that takes in one variable aka. argument and passes it through the if else statement semi looping throug the code to see if the temerature is below, above or at a comfortoble point of boiling.
+//input: i will create a function and it shall be called boilOfNoReturn. I will create an argument aka. variable an it shall be called waterTemp.
+//output: should be as expected, however i will change some stuff up and the below boilng point will put the "No one is boiling no lobster at 42", at above boiling point it will say: "350 is a little too hot for a jacuzi". And the boiling will say: "212 is just right, dinner will be ready soon."
+//process: This function is not self invoking, it will require an input.
+
+const boilOfNoReturn = (waterTemp) => {
+    if (waterTemp <= 42) {
```

Comment:
Love the messages! Quick note, you want this one to be `waterTemp < 212` since you're checking for all temperatures below 212.